runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobServerDriver.java

```diff
@@ -81,6 +75,23 @@ private static void printUsage(CmdLineParser parser) {
     System.err.println();
   }
 
+  private static ServerConfiguration parseServerConfiguration(String[] args) {
+    ServerConfiguration configuration = new ServerConfiguration();
```

Comment:
nit: is there a reason why you want the contents of `parseServerConfiguration` to be outside of `fromParams`, the redirection reduces the readability of the code.

Reply:
No, I think it was a intermediate implementation which made me extract it as a function.
Will revert it back